### ISSUE TITLE
Incompatibility with LiteLoader.

### ISSUE THREAD

**Issue by RyzenFromFire** [iLexiconn/LLibraryLegacy#70]
As the title says. Posted about it on MC Forums on the LiteLoader post.
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1290155-liteloader-for-minecraft-1-7-10?page=238

**Comment by RyzenFromFire** [comment]
Also this is for 1.7.10.

**Comment by mattmess1221** [comment]
Simply adding an if statement to the beginning of `HookPatchManager.transform` should fix it.

``` java
public byte[] transform(String name, String transformedName, byte[] classBytes) {
    if (classBytes == null) return null;
    ClassNode classNode = ASMHelper.createClassNode(classBytes);
    ...
}
```

**Comment by iLexiconn** [comment]
Sorry, but this version of LLibrary is no longer being maintained. I'll fix it in the newer version of LLibrary.

**Comment by RyzenFromFire** [comment]
Yes, but I need a fix for 1.7.10.
If you won't, can someone else do it? 
Or at least port back to 1.7.10.
That is, once you've fixed it.

**Comment by iLexiconn** [comment]
The new version of LLibrary is for Minecraft 1.7, 1.8 and 1.9.
